apache/superset
# test: Google sheets connection RTL testing

### SUMMARY
Implementing RTL testing for the Google Sheets connection flow.

### TESTING INSTRUCTIONS
- cd into `src/views/CRUD/data/database/DatabaseModal`
- Run the command `npm run test -t index.test.jsx`
- Observe the new tests

### ADDITIONAL INFORMATION
